environment/simulation/simulation.py

```diff
@@ -159,15 +166,15 @@ def _process_events(self):
             elif event.type == KEYDOWN and event.key == K_p:
                 pygame.image.save(self._screen, "swarm_ball_simulation.png")
 
-    def redraw(self, debug=False):
+    def redraw(self):
```

Comment:
Szczerze zostawił bym debug w metodzie redraw zeby nie musialo byc tak ze musimy miec cale srodowisko w trybie debug zeby podejrzeć dany jego stan w trybie debug. Środowisko sie nie zmieni, wyłącznie sposob rysowania.

Reply:
Żeby to mieć w taki sposób to chyba najlepiej by było zrobić słownik debug trzymany w całym środowisku, a później jak chcemy jakiś fragment debugować to robimy środowisko.debug["redraw"]=True. Co sądzicie?

Reply:
Nie chcialabym odpalac czesci tylko w trybie debug, albo odpalam projekt degub albo release. To i tak jest cos bardzo zewnetrzne i w finalnej wersji nie do uzywania.

Reply:
A slownik z debugowymi metodami jest tylko po cos co ja wprowadzilam na swoje potrzeby testowe. Jesli ktos chce sobie odpalic debug metody, to zawsze mozna sie prawdziwie zdebugowac :D

Z metody to usunelam, bo zeby wrzucic debug musialabym zmieniac kod i zamiast .redraw() robic .redraw(true)